pkg/csi/service/vanilla/controller_helper.go

```diff
@@ -309,13 +309,13 @@ func getCandidateDSInTopologyForFileVolumes(ctx context.Context,
 		segments := topology.GetSegments()
 		hostMoRefs, err := common.GetHostsForSegment(ctx, segments, vcenter)
 		if err != nil {
-			return nil, logger.LogNewErrorCodef(log, codes.Internal,
-				"failed to fetch hosts belonging to topology %+v. Error: %+v", topologyArr, err)
+			log.Errorf("failed to fetch hosts belonging to topology %+v. Error: %+v", topology, err)
+			continue
```

Comment:
Is it ok to ignore failure of fetching hosts for the segment and continue to next segment?

Reply:
Yes, it is ok to ignore as the length of datastores is checked at the end during the createvolume workflow of RWM volumes